crispy_forms/templates/bootstrap4/layout/button.html

```diff
@@ -1 +1 @@
-<button {{ button.flat_attrs|safe }}>{{ button.content|safe }}</button>
+<button {{ button.flat_attrs }}>{{ button.content }}</button>
```

Comment:
Making this change results in no change to the tests added in c9da48ec0b2ebf3ad491779b8a8f47b1851cf09f